sdk/resourcemanager/hybridnetwork/armhybridnetwork/fake/artifactmanifests_server.go

```diff
@@ -0,0 +1,450 @@
+//go:build go1.18
+// +build go1.18
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// Code generated by Microsoft (R) AutoRest Code Generator. DO NOT EDIT.
+// Changes may cause incorrect behavior and will be lost if the code is regenerated.
+
+package fake
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	azfake "github.com/Azure/azure-sdk-for-go/sdk/azcore/fake"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/fake/server"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/runtime"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/to"
+	"github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/hybridnetwork/armhybridnetwork"
```

Comment:
The import is missing the major version suffix. I thought I had fixed this, taking a look...

Reply:
It looks like the generator tool modifies the imports for generated code, why is that and can it be removed? Ideally, the generator should be the "source of truth" for the generated code. At any rate, guessing there's a bug/regression in that code.

Reply:
It is a bug of generator tool when it pumps the major version if detected breaking changes. We've fixed it but has not updated all the release PR. Thanks for taking a look.